### DISCUSSION TITLE
[BUG] realESRGAN doesn't save images after being generated when using optimized mode

### DISCUSSION THREAD

**Discussion by JoshuaKimsey** [hlky/stable-diffusion#86] [General]
Edit: Should I have submitted this here as a discussion or as an issue, @hlky?

Apologies if this was the wrong repo to report this on, but since I pull from this one, seemed like the right choice.

Like the title says, I was using Optimized mode (due to resolution size) to test out an older render I had done on Waifu a few days ago and wanted to do a separate upscaling with the realESRGAN, using the copy button. That went fine, if a touch slow like expected. However, I realized that realESRGAN had not saved the file anywhere in the outputs directory tree. I check both txt2img and img2img, and it just doesn't seem to be found anywhere? I even checked the "Recent" files on my machine and it's not showing up. 

Is it just being placed in an obscure location or is it truly failing to save the file? There's no output in the terminal for this either.

Also of note, I'm not sure if this is a bug or not, but despite not turning on `jpg samples` I still see it saved the grid image for the single render as a jpg image. Is that the default option for grid images now, because the Sample images are still png's?

**Comment by hlky** [comment]
The standalone tabs don't save atm, needs to be implemented, workaround is to just right click save the image

**Reply by JoshuaKimsey** [reply]
Gotcha, yeah I was just fixing to add an addendum saying it wasn't saving at all. Good to know.

**Comment by hlky** [comment]
Oh and bugs are fine as issues, I just wanted feature requests to be discussions to keep issues tidier
